### PR TITLE
More css component tailwind work

### PR DESCRIPTION
**Pull request by brdunfield**
Ported:
- conditional-radios.scss
- live-search.scss
- page-footer.scss
- preview-pane.scss
- textbox.scss

### CODE REVIEW

File: app/templates/components/live-search.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     {%- set search_label = label or form.search.label.text %}
     {% if show %}
       <div data-module="autofocus">
-        <div class="live-search js-header" data-module="live-search" data-targets="{{ target_selector }}">
+        <div class="live-search hidden js-header" data-module="live-search" data-targets="{{ target_selector }}">
```
Comment (AntoineAugusti) [inline app/templates/components/live-search.html:12]: Curious about the `hidden` class in an `if show` conditional

**Review by AntoineAugusti — APPROVED** [review]
A quick question, otherwise LGTM

File: app/assets/stylesheets/components/live-search.scss [hunk 2]
```diff
@@ -1,20 +0,0 @@
-input[type=search] {
-  // overrides this nasty global from GOV.UK template
-  -moz-box-sizing: border-box;
-  -webkit-box-sizing: border-box;
-  box-sizing: border-box;
-}
-
-.live-search {
-
-  display: none;
```
Comment (brdunfield) [inline app/assets/stylesheets/components/live-search.scss:10]: This corresponds to tailwind's "hidden" class. In response to your question @AntoineAugusti

File: app/assets/stylesheets/tailwind/style.css [hunk 3]
```diff
@@ -241,4 +243,12 @@ label {
   .research-mode {
     @apply font-bold inline-block bg-blue-govukblue text-white py-2 px-4 rounded;
   }
+
+  .js-enabled .live-search {
+    @apply block;
```
Comment (brdunfield) [inline app/assets/stylesheets/tailwind/style.css:248]: The "hidden" gets overwritten in this case here

Comment (AntoineAugusti) [reply app/assets/stylesheets/tailwind/style.css:248]: Thanks for the clarification!

**Pull request merged**